account_manager/account_manager/settings.py

```diff
@@ -0,0 +1,195 @@
+from pathlib import Path
+from datetime import timedelta # JWT
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-w8g_+)od@hs1evu3+t12a5i-)+z4u*zj&exlic)ktyp14$e=+9'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+# ------------------------------------------------------------------------------
+# 애플리케이션 설정 (Application Settings)
+# ------------------------------------------------------------------------------
+#장고 기본앱
+DJANGO_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    ]
+
+# 서드파티 앱
+THIRD_PARTY_APPS = [
+    'rest_framework',
+    'rest_framework_simplejwt',
+    'rest_framework_simplejwt.token_blacklist',
+    'drf_spectacular', # drf-spectacular을 사용하여 API 문서를 자동으로 생성하기 위해 추가함
+]
+
+# 마이 앱
+MY_APPS = [    
```

Comment:
아래로 수정해주시는게 더 적절해 보입니다.
```suggestion
LOCAL_APPS = [    
```